lib/std/meta.zig

```diff
@@ -1244,3 +1246,122 @@ test "FnPtr" {
     func = std.time.milliTimestamp;
     _ = func();
 }
+
+/// Returns a packed struct with a field layout equivalent to the layout of `T`
+/// on stage2 on little-endian systems. Integer fields are still byte-swapped
+/// according to platform endianness.
+///
+/// Requires that fields that cross byte-boundaries are byte-aligned, and that
```

Comment:
The [doc comment guidance](https://ziglang.org/documentation/master/#toc-Doc-Comment-Guidance) states "Use the word assume to indicate invariants that cause [Undefined Behavior](https://ziglang.org/documentation/master/#Undefined-Behavior) when violated."

What is the behavior, if this requirement is not satisfied?